### PR TITLE
Add commit template

### PR DESCRIPTION
**Pull request by dcramer**
This replaces the PR template with a commit template.

/cc @getsentry/team

### CODE REVIEW

File: config/commit-template [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The title
```
Comment (mattrobenolt) [inline config/commit-template]: What about:

```

Description:

Reviewers: @getsentry/team
```

My reasoning being, this commit template provides nothing. I'd be just deleting everything every time. I'd rather have prompts to fill in.

Comment (dcramer) [inline config/commit-template]: 100% not adding a 'Description:' as its already a given

I'm fine with doing Reviewers: but today we dont have a lot of value and all reviewers are signaled with @mention

Comment (mattrobenolt) [inline config/commit-template]: Yeah, I don't care what. My point is, with this template, I'm literally deleting every line and fillign in my own anyways, so the template doesn't provide much value. The template should just be prompts imo.

Maybe:

```

Changes:

Fixes:

/cc @getsentry/team
```

I dunno, I just want this to actually be useful.

**Comment by mattrobenolt** [comment]
:+1:

**Pull request merged**